src/Control/Selective.hs

```diff
@@ -196,10 +196,10 @@ branch x l r = fmap (fmap Left) x <*? fmap (fmap Right) l <*? r
 selectA :: Applicative f => f (Either a b) -> f (a -> b) -> f b
 selectA x y = (\e f -> either f id e) <$> x <*> y
 
--- | If a functor is both 'Applicative' and 'Traversable', we can implement
--- 'select' in another interesting way: the effects associated with the second
--- argument can be skipped as long as the first argument contains only 'Right's.
-selectT :: (Applicative f, Traversable f) => f (Either a b) -> f (a -> b) -> f b
+-- | For traversable functors, we can implement 'select' in another interesting
```

Comment:
```suggestion
-- | For 'Traversable' 'Functor's, we can implement 'select' in another interesting
```

Reply:
I think we tend to say just "applicative functors" in many comments already, so sticking to "traversable functors" seems more consistent.